girder/api/describe.py

```diff
@@ -139,76 +139,83 @@ class ApiDocs(object):
-          up a list of all available endpoints related to each resource type.
-          </p>
-          <p>Clicking any of those endpoints will display detailed documentation
-          about the purpose of each endpoint and the input parameters and output
-          values. You can also call API endpoints directly from this page by
-          typing in the parameters you wish to pass and then clicking the "Try
-          it out!" button.</p>
-          <p><b>Warning:</b> This is not a sandbox&mdash;calls that you make
-          from this page are the same as calling the API with any other client,
-          so update or delete calls that you make will affect the actual data on
-          the server.</p>
-        </div>
-        <div class="swagger-section">
-          <div id="swagger-ui-container"
-              class="swagger-ui-wrap docs-swagger-container">
-          </div>
-        </div>
-        <script src="${staticRoot}/built/swagger/lib/jquery-1.8.0.min.js">
-        </script>
-        <script src="${staticRoot}/built/swagger/lib/jquery.slideto.min.js">
-        </script>
-        <script src="${staticRoot}/built/swagger/lib/jquery.wiggle.min.js">
-        </script>
-        <script src="${staticRoot}/built/swagger/lib/jquery.ba-bbq.min.js">
-        </script>
-        <script src="${staticRoot}/built/swagger/lib/handlebars-1.0.0.js">
-        </script>
-        <script src="${staticRoot}/built/swagger/lib/underscore-min.js">
-        </script>
-        <script src="${staticRoot}/built/swagger/lib/backbone-min.js"></script>
-        <script src="${staticRoot}/built/swagger/lib/shred.bundle.js"></script>
-        <script src="${staticRoot}/built/swagger/lib/swagger.js"></script>
-        <script src="${staticRoot}/built/swagger/swagger-ui.min.js"></script>
-        <script src="${staticRoot}/built/swagger/lib/highlight.7.3.pack.js">
-        </script>
-        <script src="${staticRoot}/girder-swagger.js"></script>
-      </body>
-    </html>
-    """
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>${title}</title>
+    <link rel="stylesheet"
+        href="//fonts.googleapis.com/css?family=Droid+Sans:400,700">
+    <link rel="stylesheet"
+        href="${staticRoot}/lib/fontello/css/fontello.css">
+    <link rel="stylesheet"
+        href="${staticRoot}/built/swagger/css/reset.css">
+    <link rel="stylesheet"
+        href="${staticRoot}/built/swagger/css/screen.css">
+    <link rel="stylesheet"
+        href="${staticRoot}/built/swagger/docs.css">
+    <link rel="icon"
+        type="image/png"
+        href="${staticRoot}/img/Girder_Favicon.png">
+    <style type="text/css">
+.response_throbber {
+  content:url("${staticRoot}/built/swagger/images/throbber.gif");
+}
+    </style>
+  </head>
+  <body>
+    <div class="docs-header">
+      <span>Girder REST API Documentation</span>
+      <i class="icon-book-alt right"></i>
+      <div id="g-global-info-apiroot" class="hide">${apiRoot}</div>
```

Comment:
This doesn't appear to be hidden correctly, maybe need to just directly set `display: none`.

![screen shot 2015-04-05 at 8 04 06 pm](https://cloud.githubusercontent.com/assets/555959/6999225/07a79944-dbcf-11e4-967a-2a42654d2489.png)